addons/hr_skills_slides/models/slide_channel.py

```diff
@@ -43,3 +43,43 @@ def _recompute_completion(self):
                         'channel_id': channel.id
                     })
         return res
+
+    def _send_completed_mail(self):
+        super()._send_completed_mail()
+        for scp in self:
+            if self.env.user.employee_ids:
+                msg = _('The employee has completed the course <a href="%(link)s">%(course)s</a>',
+                    link=scp.channel_id.website_url,
+                    course=scp.channel_id.name)
+                self.env.user.employee_ids\
+                    .filtered(lambda e: e.company_id == self.env.company)\
+                    .message_post(body=msg)
```

Comment:
You can use `employee_id` instead, it will always be the one of the current company :-) [1]

```suggestion
                self.env.user.employee_id.message_post(body=msg)
```

[1] : https://github.com/odoo/odoo/blob/6436c209d6f601e5414ed800dd54d244b3f1b289/addons/hr/models/res_users.py#L258